Formula/erlang_ls.rb

```diff
@@ -0,0 +1,19 @@
+class ErlangLs < Formula
+  desc "Erlang Language Server"
+  homepage "https://erlang-ls.github.io/"
+  url "https://github.com/erlang-ls/erlang_ls/archive/refs/tags/0.41.2.tar.gz"
+  sha256 "84aa38af9682186596a5d4c65682f553e964059eb81c4d00607b8ead97305def"
+  license "Apache-2.0"
+
+  depends_on "rebar3" => :build
+  depends_on "erlang"
+
+  def install
+    system "make"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/erlang_ls", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
You can look at other language servers for example tests.

Reply:
Done! I copied the test from https://github.com/Homebrew/homebrew-core/blob/master/Formula/yaml-language-server.rb.